docs/toc.yml

```diff
@@ -9,7 +9,7 @@
 - name: .NET Desktop Guide - WPF
   href: desktop-wpf/
 - name: C# Guide
-  href: csharp/
+  href: csharp/index.yml
```

Comment:
Not sure this change was necessary.

Reply:
@gewarren Without this change, I was getting build warnings. A similar change was made for the ML.NET landing page.